### PR TITLE
Explore/setup

### PR DESCRIPTION
**Pull request by charblanc**

### CODE REVIEW

**Comment by charblanc** [comment]
My merge with master is what made it fail ^^ ?

**Comment by ethanbb** [comment]
There's a merge conflict in package.json lol

**Comment by charblanc** [comment]
fixed that lol .... my b.

File: server/explore.js [hunk 1]
```diff
@@ -1,79 +0,0 @@
-/* explore.js
```
Comment (ethanbb) [inline server/explore.js:1]: What happened to this code?

Comment (charblanc) [reply server/explore.js:1]: good catch! I think i deleted the file in my branch and it didn't re add it on merge.

Comment (charblanc) [reply server/explore.js:1]: Added your function into explore_controller.js

Comment (ethanbb) [reply server/explore.js:1]: OK cool

File: server/models/user.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ const userSchema = new Schema({
   groups: [{ type: Schema.Types.ObjectId, ref: 'Group' }],
   usersIFollow: [{ type: Schema.Types.ObjectId, ref: 'User' }],
   usersFollowingMe: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  exploreNumber: Number,
-  numExplorations: Number,
+  exploreNumber: { type: Number, default: 0 },
+  numExplorations: { type: Number, default: 20 },
```
Comment (ang-zhu) [inline server/models/user.js]: So this is just assuming that explore won't serve anything until they've annotated 20 articles? Or assuming that we will be able to pull in 20 scores from FB?

Comment (ang-zhu) [inline server/models/user.js]: So this is just assuming that explore won't serve anything until they've annotated 20 articles? Or assuming that we will be able to pull in 20 scores from FB?

**Review by ethanbb — APPROVED** [review]

File: server/models/user.js (hunk 2 above)
Comment (charblanc) [reply server/models/user.js]: ah.

Neither - the idea is that if they use the explore plugin/algorithm and populate with facebook data, then we say "numExplorations" is 20 so after they've annotated 20 articles, their exploreNumber will be 50% from facebook data and 50% from notist data, so therefore the facebook data will become less and less important. BUT, Now that I think about it, it doesn't make sense for default to be 20... so I'll change.

**Pull request merged**